test/features/show.hide.timeout.feature

```diff
@@ -13,7 +13,9 @@ Feature: Present / Not present
   Scenario: I wait step should wait for specified amount of milliseconds
     Given I open "http://localhost:9000"
     Then txtHeader from main page should be present
+    Then txtHeader from main page has text "Header"
```

Comment:
Please write a separate `Scenario:` for each step you add

Reply:
Done